src/utils/registerPlugin.ts

```diff
@@ -0,0 +1,19 @@
+import { ProcessOpenClientResponseContext } from './processOpenClientResponse'
+
+export type ProcessUnsealedDataPluginFunction = (context: ProcessOpenClientResponseContext) => void | Promise<void>
+export enum PluginType {
```

Comment:
In the past, we have [discussed](https://github.com/fingerprintjs/fingerprintjs-pro-server-api-node-sdk/pull/104#discussion_r1634862997) using type literals instead of enums for limited sets of values like this.

Enums can be slightly annoying in that: 
* You need to import an extra thing to use it
* If you ever need to read that value from `.env` or something like that, you need extra code to map it onto the enum

I am not sure those things really apply here, but I wondered what advantages there are to using an enum instead of just

`type PluginType = 'processOpenClientResponse'` 

And the user passing the type-checked string directly?

Reply:
Refactored by this commit: https://github.com/fingerprintjs/fingerprint-pro-fastly-proxy-integration-example/pull/5/commits/9ed332cbca6e69231b189e0160f88247d326e704